src/actions/breakpoints/index.js

```diff
@@ -352,7 +350,7 @@ export function addBreakpointAtLine(line: number) {
     const state = getState();
     const source = getSelectedSource(state);
 
-    if (!source || isEmptyLineInSource(state, line, source.id)) {
```

Comment:
lets keep empty lines. Even though it is not available now it will be nice to fix it up tomorrow.

Reply:
That made the tests fail though?

Reply:
Should be good now :)